cypress/integration/orders/orders.PoSearch.spec.js

```diff
@@ -48,4 +49,16 @@ describe('orders: Test PO search', () => {
         Orders.checkSearchResults(orderNumber);
       });
   });
+  it('C6718 Test the PO filters [except tags]', { tags: [TestType.smoke] }, () => {
```

Comment:
"Test each filter, separately or in combination with others. You may need to create new POs or edit existing POs to add various details so that you can test the filters." - test should be redesigned. See best practices in https://wiki.folio.org/display/DQA/FOLIO+test+cases+guideline